bot/exts/utilities/issues.py

```diff
@@ -139,7 +139,7 @@ async def fetch_issues(
             log.trace(f"PR provided, querying GH pulls API for additional information: {pulls_url}")
             async with self.bot.http_session.get(pulls_url) as p:
                 pull_data = await p.json()
-                if pull_data["draft"]:
+                if "draft" in pull_data and pull_data["draft"]:
```

Comment:
How about using ```pull_data.get("draft")``` instead? 

Same review as [979](https://github.com/python-discord/sir-lancebot/pull/979#pullrequestreview-829455026)

Reply:
hm, this commit shouldn't even be on this pr.

@evgriff this change is out of scope of this pr, so you'll need to remove it. you can do it with either resetting your HEAD and then forcepushing, or making a commit which will revert this commit.